core/src/mindustry/content/TechTree.java

```diff
@@ -162,6 +162,10 @@ public void load(){
 
                                             node(Items.plastanium, with(Items.titanium, 10000, Items.silicon, 10000), () -> {
                                                 node(plastaniumCompressor, () -> {
+                           
+                                                node(plastaniumPolymerizer, () -> {
+                                                
+                                                });
```

Comment:
the indentation was wrong:
```suggestion
                                                    node(plastaniumPolymerizer, () -> {
                                                
                                                    });
```